cmd/dkg.go

```diff
@@ -0,0 +1,52 @@
+// Copyright © 2022 Obol Labs Inc.
+//
+// This program is free software: you can redistribute it and/or modify it
+// under the terms of the GNU General Public License as published by the Free
+// Software Foundation, either version 3 of the License, or (at your option)
+// any later version.
+//
+// This program is distributed in the hope that it will be useful, but WITHOUT
+// ANY WARRANTY; without even the implied warranty of  MERCHANTABILITY or
+// FITNESS FOR A PARTICULAR PURPOSE. See the GNU General Public License for
+// more details.
+//
+// You should have received a copy of the GNU General Public License along with
+// this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package cmd
+
+import (
+	"context"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/pflag"
+
+	"github.com/obolnetwork/charon/dkg"
+)
+
+func newDKGCmd(runFunc func(context.Context, dkg.Config) error) *cobra.Command {
```

Comment:
I don't understand the goal of passing runFunc instead of calling directly dkg.Run from here. Are we going to use this parameter with something different at some point?

Reply:
yeah, this allows testing the configuration flags

Reply:
ok